src/Trixi.jl

```diff
@@ -156,6 +156,7 @@ export GradientVariablesPrimitive, GradientVariablesEntropy
 
 export flux, flux_central, flux_lax_friedrichs, flux_hll, flux_hllc, flux_hlle,
        flux_godunov,
+       flux_upwind,
```

Comment:
@ranocha not sure if this name is too generic, what do you think? It *is* an upwind flux, so it wouldn't be wrong per se...

Reply:
Isn't it simply the Godunov flux that we call `flux_godunov`?

Reply:
You're probably right. @lchristm can you rename the flux accordingly please?

Reply:
Done in bd926117.